Project-I-Social-Card/social-card/src/App.css

```diff
@@ -1,4 +1,61 @@
-/* 
-  Write your styles/CSS for App.js here and import into the component that you need 
+/*
+  Write your styles/CSS for App.js here and import into the component that you need
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+
```

Comment:
good job with your CSS. It's always good to get in some practice to keep your old skills still sharp. Code is super clean and easy to read too